rust/feature-flags/src/api/types.rs

```diff
@@ -55,11 +55,11 @@ impl LegacyFlagsResponse {
             feature_flag_payloads: response
                 .flags
                 .iter()
-                .map(|(key, flag)| {
-                    (
-                        key.clone(),
-                        flag.metadata.payload.clone().unwrap_or(Value::Null),
-                    )
+                .filter_map(|(key, flag)| {
```

Comment:
Nice to see `filter_map` built in. In C#, I'm always writing a `WhereNotNull(...)` method in each new project.

Reply:
yeah, fluent interfaces in Rust have some great tools. 

Technically, `Null` and `None` are different in Rust though, so this method _will_ filter out flags with no payload, but if the payload is `Null` (which is technically a valid JSON payload), it'll remain (by design).